Server/File/IFileContentProvider .cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Karma.Server.Models;
+
+namespace Karma.Server.File
+{
+    public interface IFileContentProvider
+    {
+        public List<Item> ReadFromFile ();
```

Comment:
By default interface members are public(unless you specify it to be private etc.) so public is not needed here

Reply:
Makes sense, fixed it.